joomla/joomla-cms
# [4.0] Change tags "list all" behavior

Pull Request for Issue # https://github.com/joomla/cassiopeia/issues/209 & #31388 

### Summary of Changes
Changes flow of tags in list all view to columnar to match options.
Supports responsive flow.

### Testing Instructions
If your using the test data sample then there is an "all tags" item otherwise...
Create some tags.
Create a menu item of type "List All Tags".
Play with the option "Number of Columns". ( in menu item )
apply pr
run npm build:css
verify that the number of columns matches the option chosen.
works based on # of columns.  If columns = 0 then its flex flow so everything just flows & sizes automatically, if columns = 1 it's vertical layout, else its columnar layout.

### Actual result BEFORE applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/99383129-10acdd80-2893-11eb-92e9-9450d04b0fe4.png)



### Expected result AFTER applying this Pull Request
![Firefox](https://user-images.githubusercontent.com/1850089/99377700-223eb700-288c-11eb-818d-66b737059fba.png)
![Chrome](https://user-images.githubusercontent.com/1850089/99383054-f8d55980-2892-11eb-9703-8efec8f8d5c8.png)
![tags](https://user-images.githubusercontent.com/1850089/99384075-81082e80-2894-11eb-8d19-6a613521271b.jpg)


### Documentation Changes Required
works based on # of columns.  If columns = 0 then its flex flow so everything just flows & sizes automatically, if columns = 1 it's vertical layout, else its columnar layout.
